packages/ui-text-area/src/TextArea/index.tsx

```diff
@@ -318,31 +282,29 @@ class TextArea extends Component<TextAreaProps> {
         defaultValue={defaultValue}
         placeholder={placeholder}
         ref={(textarea, ...args) => {
-          // @ts-expect-error ts-migrate(2339) FIXME: Property '_textarea' does not exist on type 'TextA... Remove this comment to see the full error message
           this._textarea = textarea
-          // @ts-expect-error ts-migrate(2532) FIXME: Object is possibly 'undefined'.
-          textareaRef.apply(this, [textarea].concat(args))
+          if (typeof textareaRef === 'function') {
+            textareaRef.apply(this, [textarea, ...args])
```

Comment:
I have no clue why they wrote it like this XD I really doubt that the `ref=""` would pass any more args, so there is nothing else to pass to the textareaRef other than the node. I would refactor and simplify it, but ask someone for second opinion on this :)

Reply:
I did not refactor it by choice, so it can be as minimal chance to change the behaviour as possible. Weird for sure :D

Reply:
Well alright then (: